lineupStyle.css

```diff
@@ -0,0 +1,101 @@
+.Featured h2 {
+  font-family: "lato", sans-serif;
+  font-size: 25px;
+  line-height: 40px;
+  font-weight: 400;
+  color: black;
+  text-align: center;
+  margin-top: 20px;
+}
+
+.underLine {
+  border-bottom: 2px solid #ec5242;
+  width: 15%;
+  padding-top: 10px;
+  margin: 0 auto;
+  margin-bottom: 20px;
+}
+
+.speaker {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px;
+  transition: transform 0.3s ease;
+}
+
+.speaker:hover {
+  transform: scale(1.1);
+}
+
+.speakerInfo {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10px;
+}
+
+.speakerInfo h5 {
+  font-family: "lato", sans-serif;
+  font-size: 15px;
+  line-height: 20px;
+  font-weight: 700;
+  color: black;
+  text-align: left;
+  margin-top: 20px;
+}
+
+.about {
+  font-family: "lato", sans-serif;
+  font-size: 15px;
+  line-height: 15px;
+  font-weight: 700;
+  color: #ec5242;
+  text-align: left;
+  margin-top: 20px;
+}
+
+.description {
+  font-family: "lato", sans-serif;
+  font-size: 12px;
+  line-height: 20px;
+  font-weight: 700;
+  color: black;
+  text-align: left;
+  margin-top: 20px;
+}
+
+.moreBtn {
+  width: 90%;
+  height: 50px;
+  margin: 10px auto;
+  font-size: 23px;
+  background-color: white;
+  border: 1px solid #d3d3d3;
+  transition: transform 0.3s ease;
+}
+
+.moreBtn::after {
+  content: "\25BE";
+  color: #ec5242;
+  font-size: 40px;
+}
+
+.moreBtn:hover {
+  transform: scale(1.1);
+  color: #ec5242;
+}
+
+/* Desktop */
+@media screen and (min-width: 768px) {
+  .speak {
+    display: grid;
+    grid-template-columns: auto auto;
+  }
+
+  .speaker {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    margin: 1.2rem 0;
+  }
```

Comment:
- [OPTIONAL] Good job so far, :clap: , however,  Please ensure that your speakers' layout aligns correctly with the design to match the design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student Speakers options](https://github.com/DevKaranJ/Capstone-project-Music-Festival-Website/assets/43528594/82866dde-4642-4ddf-8a24-28de06bf9859)


</td>
      <td>

![Design Speakers Options](https://github.com/DevKaranJ/Capstone-project-Music-Festival-Website/assets/43528594/4f0b32cc-b650-4df8-97ee-0cb1ba77f026)

</td>
    </tr>
  </table>